docs/changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ### v0.8.0 (master)
 
+**Highlights**
+- Support audio recognition
```

Comment:
does this description accurate? 
`Support action recognition with audio modality`

Reply:
`Support video recognition with audio modality` maybe better.